schema.sql

```diff
@@ -1,5 +1,11 @@
 /* Database schema to keep the structure of entire database. */
 
```

Comment:
### Optional
- [ ] It would be better if we capitalize our commit messages, nice way of commits :+1: